CONTRIBUTING.md

```diff
@@ -21,6 +21,18 @@ Due to legal reasons, all contributors must sign a contributor agreement, either
 * Tag PRs with either `toward: #<bug>` or `fixes: #<bug>` to help establish context for why a change is happening
 * Commit messages follow [Chris Beam's awesome commit message style guide](http://chris.beams.io/posts/git-commit/)
 
+### Go error reporting
+
+* For problems that are conceptually `asserts`, meaning that the program cannot meaningfully continue, use the following calls:
+	* d.Chk.Error()
+	* d.Chk.NoError()
+	* d.Chk.True()
+	* d.Chk.False()
+* For problems that might occur during correct processing, you can choose the Go-style of returned errors, or use the following to 'raise' a 'catchable' error (see [go/d/try.go](https://godoc.org/github.com/attic-labs/noms/go/d)):
+	* d.PanicIfError()
```

Comment:
You might want to add two additional points:
In general, for Public API in Noms, we use the Go-style of returned errors by default.
* There must be an overriding rational for using the Exception-style. One reason to use the Exception style is that the current code doesn't know how to proceed and needs to panic, but we want to signal that a calling function somewhere up the stack might be able to recover from the panic and continue on.
* Test code rarely, if ever, needs or should use these Exception-style functions